e2e/tsconfig.json

```diff
@@ -1,11 +1,19 @@
 {
   "compilerOptions": {
-    "target": "es6",
```

Comment:
This `tsconfig` change is worth moving to a separate commit with the description you provided in the PR details.